react-ui/public/service.js

```diff
@@ -656,8 +658,8 @@ function processItemSentClicked(entity) {
     updateListing(id, { "status": "completed", 'notification': true });
     getListingById(id, function (listing) {
         //MDS.log(`Listing found: ${JSON.stringify(listing)}`);
-        notification(`Your purchase ${listing.title} has been sent by @${listing.created_by_name} and is now in transit.`);
-        var message = `Your purchase ${listing.title} has been sent by @${listing.created_by_name} and is now in transit.`
+        notification(`Your purchase '${listing.title}' has been sent by @${listing.created_by_name} and is now in transit.`);
+        var message = `Your purchase ''${listing.title}' has been sent by @${listing.created_by_name} and is now in transit.`
```

Comment:
this looks like there are one double quotes and one single quote

Reply:
hahaha. i agree.